### PR TITLE
Issue1389 crash course package

### LINKED ISSUE #1389: Create separate package within IDEAS.Fluid.Tutorial for Modelica crash course examples
The Crash course examples used to be the only ones inside the IDEAS.Fluid.Tutorial package, but the latest IBPSA merge (PR #1383) added a SimpleHouse package to the Tutorial package. The Crash course examples should therefore be collected in a seperate package as well (including some documentation).

### PR DESCRIPTION
**Pull request by LoneMeertens**
fixes #1389

### CODE REVIEW

**Comment by lucasverleyen** [comment]
@LoneMeertens , thank you for creating the pull request! The tests are indeed failing because the paths have changed. Let's change that once the structure of the new package has been finalised. I wonder if we could create a new package DetailedHouse to collect Example 1 - 10. Now, you have divided the examples across the exercises of the crash course, but for external people, this is not that relevant. We could follow the same approach as SimpleHouse: collecting all steps in the same package. SimpleHouse uses a simple RC model. DetailedHouse uses a detailed IDEAS building model. What do you think?

**Comment by LoneMeertens** [comment]
@lucasverleyen Thank you for the feedback! Creating a DetailedHouse package to collect Examples 1–10 makes sense and aligns well with the SimpleHouse approach. I’ll work on modifying the structure accordingly.

**Comment by lucasverleyen** [comment]
I was also thinking about the SimpleHouse.mo model that is in the original IBPSA library at the same level as the Tutorial package. I would suggest keeping the model there because we'll probably not change that model soon. The less we change compared to IBPSA, the easier it is to merge the latest changes of IBPSA into IDEAS.

File: IDEAS/Examples/Tutorial/CrashCourse/Exercise3/Example1.mo [hunk 1]
```diff
@@ -76,31 +75,31 @@ model Example1 "First example model containing one zone"
         origin={70,10})));
 equation
   connect(outerWall.propsBus_a, zone.propsBus[1]) annotation (Line(
-      points={{-45,12},{-38.5,12},{-38.5,15.7143},{-20,15.7143}},
+      points={{-45,12},{-38.5,12},{-38.5,13.1429},{-20,13.1429}},
```
Comment (lucasverleyen) [inline IDEAS/Examples/Tutorial/CrashCourse/Exercise3/Example1.mo:78]: I see quite some unintended changes, due to different Dymola versions. Sometimes Dymola changes other lines in the text if you save a model. However, it is a good practice to check the lines of code that you commit and only commit the lines that you really want to change. You can discard all other changes. 

As there are many, and you will change the structure anyway, I propose to start from a new branch, which is easier than reverting these changes.

**Pull request closed without merging**